Content.Server/Light/Components/PoweredLightComponent.cs

```diff
@@ -245,21 +245,30 @@ public void UpdateLight()
                     }
                     else
                     {
-                        _currentLit = false;
+                        SetLight(false);
                         _appearance?.SetData(PoweredLightVisuals.BulbState, PoweredLightState.Off);
                     }
                     break;
                 case LightBulbState.Broken:
-                    _currentLit = false;
+                    SetLight(false);
                     _appearance?.SetData(PoweredLightVisuals.BulbState, PoweredLightState.Broken);
                     break;
                 case LightBulbState.Burned:
-                    _currentLit = false;
+                    SetLight(false);
                     _appearance?.SetData(PoweredLightVisuals.BulbState, PoweredLightState.Burned);
                     break;
             }
         }
 
+        private void SetLight(bool value, Color color = default)
+        {
+            _currentLit = value;
+
+            if (!Owner.TryGetComponent(out PointLightComponent? pointLight)) return;
+            pointLight.Enabled = value;
+            pointLight.Color = color;
```

Comment:
Why does enabling/disabling light reset its color to default?

Reply:
i suppose it doesnt really matter since the color always gets sent on enable. the argument could be made tho that one could save a little bandwidth here, altough i believe its minimal at best

Reply:
It's getting dumped in the state regardless of whether it's reset but I changed it in case some admin VVs it on I guess.